theme.json

```diff
@@ -785,10 +785,10 @@
 		"spacing": {
 			"blockGap": "1.2rem",
 			"padding": {
-				"bottom": "0vh",
-				"left": "6vw",
-				"right": "6vw",
-				"top": "0vh"
+				"bottom": "0",
```

Comment:
it's better if we don't define as 0 the vertical padding, else we have a [bug with the nav menu](https://github.com/WordPress/gutenberg/pull/53725)